src/Components/Solution.js

```diff
@@ -0,0 +1,24 @@
+import Letter from "./Letter";
+import React, { Component } from 'react';
+
+function displayUnderscores(){
+    let underscore = [];
+    for (let i=0; i<4; i++){
```

Comment:
instead of the `4`, better use `this.props.solution.word.length` to make it dynamic. 
also usually we do that with `map` inside the `return` statement (inside the `div`)